### DISCUSSION TITLE
Deploy/Cleanup

### DISCUSSION THREAD

**Discussion by LevT2** [Microsoft/WSLab#526] [Q&A]
Is it correct that I can run no mоre than one lab simultaneously?
If the answer is no, is there a supported and straightforward procedure to intertwin lifecycles of several labs?

**Comment by LevT2** [comment]
Up to my today's understanding, the concrete lab is indicated by default by the directory name (if the LabPrefix is not defined)
Moreover, it is a user responsibility to not mess with these lab identifiers - otherwise, you have to do manual cleanup at minimum.

**Reply by jaromirk** [reply]
Exactly! Just remove prefix (it will use directory name), or keep prefix unique for each lab. You can clone directories(when lab is not deployed) , that's fine

**Reply by LevT2** [reply]
@jaromirk 
 I think your intention was to move labs between laptops etc. But in my use case it would be better not to clone ParentDisks.

Last few weeks I wasted a lot of time trying to work directly with the cloned sources, and sometimes I got success deploying labs and cleaning them up. But I couldn't reproduce this behavior with confidence.

**Reply by jaromirk** [reply]
When I was developing MSLab (back then it was ws2016lab), I made it with intention to have hydrated lab (state before running deploy) on USB SSD and use it for workshops - before starting workshop I just asked participants to copy one directory and that was it. So it should work. Feel free to ping me at teams to discuss it (Jaromir.kaspar@dell.com)
